python/tvm/relay/backend/contrib/ethosu/te/pooling.py

```diff
@@ -110,18 +110,21 @@ def pooling_compute(
     padding = [int(v) for v in padding]
     stride_h, stride_w = [int(v) for v in strides]
     pool_shape_h, pool_shape_w = [int(v) for v in pool_shape]
+    ifm_channels = ofm_channels if pooling_type != "SUM" else ifm.shape[-1]
```

Comment:
Won't block on this as the functionality remains the same, but wondering if `ofm_channels` ever made sense here and whether we can just use `ifm.shape[-1]` for all pooling types?